Homework11/src/Taming.java

```diff
@@ -0,0 +1,4 @@
+public interface Taming<T,W> {
+
+        void accept(T t,W w);
+}
```

Comment:
```suggestion
public interface Taming<T,W, R> {

        R accept(T t,W w);
}
```
и желательно еще добавить аннотацию, что это именно функциональный интерфейс